test/runner/requirements/sanity.txt

```diff
@@ -1,4 +1,5 @@
 cryptography
+fortiosapi
```

Comment:
Please remove this line.

This is only for what's common across all modules, notice no boto3 which is used by all AWS modules